prototype 2/mediapipe_handtracking_demo/mediapipeht.py

```
@@ -1,27 +1,59 @@
-mp_hands = mp.solutions.hands
+
+BaseOptions = mp.tasks.BaseOptions
+HandLandmarker = mp.tasks.vision.HandLandmarker
+HandLandmarkerOptions = mp.tasks.vision.HandLandmarkerOptions
+VisionRunningMode = mp.tasks.vision.RunningMode
+
+MODEL_PATH = "../hand_landmarker.task"
+
+current_landmarks = None
+
+def hand_callback(result, output_image, timestamp_ms):
```

Ruff (0.14.10)

Unused function argument: `output_image`

(ARG001)